src/components/edit-file.js

```diff
@@ -1,51 +1,64 @@
 import React, { Component } from 'react'
 import { connect } from 'react-redux'
 import { Button, Icon } from 'semantic-ui-react'
-import { editFile } from '../actions/index'
+
 import EditModal from './edit-modal'
+import { editFileName } from '../actions/fileAction'
+import { getFolder } from '../actions/folderActions'
 
 class EditFile extends Component {
-  constructor(props) {
+  constructor (props) {
     super(props)
     this.state = {
       showModal: false
     }
   }
-  componentDidUpdate(prevProps) {
-    const { selectedData } = this.props
-    if (
-      JSON.stringify(prevProps.selectedData) !== JSON.stringify(selectedData)
-    ) {
-      this.setState({
-        fileName: selectedData.fileName,
-        isPublic: selectedData.isPublic
-      })
-    }
-  }
-  handleEdit = () => {
+
+  showModal = () => {
     this.setState({
```

Comment:
why make separate function for just one line of command? use inline functions in modal